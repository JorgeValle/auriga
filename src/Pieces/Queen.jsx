import React from 'react';

// Components should represent one piece of data model
class Queen extends React.Component {

  render() {
  
    return (
      <i class="fas fa-chess-queen"></i>
    );
  }
}

export default Queen;