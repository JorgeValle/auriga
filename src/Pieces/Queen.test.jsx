import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import Queen from './Queen.jsx';

// We shallow render to make sure changes in children don't cause cascading fails

configure({
  adapter: new Adapter()
});

describe('<Queen />', () => {

  it('shallow renders', () => {
    const wrapper = shallow(<Queen />);
  });

});

