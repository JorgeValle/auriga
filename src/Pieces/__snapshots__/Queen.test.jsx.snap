// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Queen /> matches snapshot 1`] = `ShallowWrapper {}`;
